File: internal/content/testdata/bench/rust/nc-plus-one.rs
fn plus_one(digits: Vec<i64>) -> Vec<i64> {
    let mut digits = digits;
    let n = digits.len();
    for i in (0..n).rev() {
        if digits[i] < 9 {
            digits[i] += 1;
            return digits;
        }
        digits[i] = 0;
    }
    let mut result = vec![1i64];
    result.extend(digits);
    result
}

File: internal/content/testdata/bench/csharp/nc-plus-one.cs
public class Solution {
    public long[] plus_one(long[] digits) {
        for (int i = digits.Length - 1; i >= 0; i--) {
            if (digits[i] < 9) {
                digits[i]++;
                return digits;
            }
            digits[i] = 0;
        }
        long[] result = new long[digits.Length + 1];
        result[0] = 1;
        return result;
    }
}
